src/mail/mail.service.ts

```diff
@@ -0,0 +1,21 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable } from '@nestjs/common';
+import { SentMessageInfo } from 'nodemailer';
+
+@Injectable()
+export class MailService {
+  constructor(private mailerService: MailerService) {}
+
+  async sendAccountActivationEmail(
+    userEmail: string,
+    token: string,
+  ): Promise<SentMessageInfo> {
+    const url = `http://localhost:3000/auth/activate-account/?token=${token}`;
```

Comment:
change it to `baseUrl` from env 
```suggestion
    const url = `${baseUrl}/auth/activate-account/?token=${token}`;
```